gym_graph_traffic/envs/intersection.py

```diff
@@ -38,14 +50,15 @@ def segment_draw_coords(self, length, to_side):
 
 class FourWayNoTurnsIntersection(Intersection):
```

Comment:
We shouldn't add turns to FourWayNoTurnsIntersection :) It would be perfect to create a new class with turns, and leave this one unchanged (with regard to observable behaviour at least) to let the user come back to the old behaviour, but it probably is not trivial due to tight coupling with Segments logic.